### DISCUSSION TITLE
How To Make Sound Fill The Entire Room?

### DISCUSSION THREAD

**Discussion by tedtheminer** [mozilla/hubs#2845] [General Questions & Discussion] [answered]
I have been trying to make a concert hall with a screen at the front that could play videos and realised that the sound doesn't reach the back so I tried increasing the volume and other things but still can't figure it out. Is there a way to have the audio from a video panel fill the entire room?

**Comment by ocularvr** [comment, marked as answer]
Easy one,when in Spoke,  just go to the video you want to use, then go to its AUDIO TYPE drop down list and change from PANNERNODE to STEREO. This will ensure that the sound level is the same in your hall everywhere - set the master volume to the desired level, very important.

**Comment by sobralfreenat** [comment]
Although the answer is fine for some use cases, it will kill spatial localization. Better is to set the pannermode correctly considering the size of your room.
